akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -126,6 +126,18 @@ akka.serialization.jackson {
   # }
   json-write-features {}
 
+  # Configuration of the JsonFactory Visibility.
+  # See com.fasterxml.jackson.annotation.PropertyAccessor
+  # and com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility
+  # Enum values, for example:
+  #
+  # json-write-features {
+  #   FIELD = PUBLIC_ONLY
+  # }
+  visibility {
+    FIELD = ANY
+  }
```

Comment:
Documentation does not correspond configuration, or it is confusing...
I looked into the tests, and conf differs from this file again 😕

Reply:
@PromanSEW, the comment introduces an example while the default value is a different thing. I will rewrite the comment to make it less confusing.